pkg/intcode/intcode.go

```diff
@@ -204,10 +170,13 @@ func (*stop) run(p *program, arg1, _, _ int) bool {
 	}
 
 	// normal stop
+	if p.maxOp < Output {
+		p.outputs <- p.instructions[0]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [avoid magic numbers like '0', create a named constant for it](https://app.codacy.com/manual/afarbos/aoc/pullRequest?prid=5498712)